recipes/tcl/all/conanfile.py

```diff
@@ -137,7 +139,8 @@ def _build_nmake(self, targets):
     @functools.lru_cache(1)
     def _configure_autotools(self):
         autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
-        yes_no = lambda v: "yes" if v else "no"
+        def yes_no(v):
```

Comment:
please, avoid updating methods which are working already, like this lamdba which largely used in CCI (around 470 occurrences)